db/migrate/20230615084630_add_last_active_role_to_people.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Hitobito AG. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
```

Comment:
maybe make use of newly introduced role scopes in this migration

Reply:
thanks, @carlobeltrame. Scopes change (with this PR, for example) and the App relies on the migrations which rely on the Database, not the other way around.